packages/lib-classifier/src/store/Workflow/WorkflowConfiguration/WorkflowConfiguration.js

```diff
@@ -5,7 +5,8 @@ const WorkflowConfiguration = types.model({
   enable_switching_flipbook_and_separate: types.optional(types.boolean, false),
   hide_classification_summaries: types.optional(types.boolean, false),
   multi_image_mode: types.optional(types.enumeration('multiImageMode', ['flipbook', 'separate']), 'flipbook'),
-  subject_viewer: types.maybe(types.enumeration('subjectViewer', ['lightcurve', 'multiFrame', 'subjectGroup']))
+  subject_viewer: types.maybe(types.enumeration('subjectViewer', ['lightcurve', 'multiFrame', 'subjectGroup'])),
+  subject_viewer_config: types.frozen()
```

Comment:
I think the type here either needs to be `types.maybe(types.frozen())` which defaults it to undefined or provide a default value as `types.frozen({})`.

Reply:
👍 Solid suggestion - I've made the change to `types.frozen({})`